galaxy_ng/app/migrations/0019_pulp_container_access_policies.py

```diff
@@ -0,0 +1,193 @@
+from django.db import migrations
+import django_lifecycle.mixins
+import galaxy_ng.app.access_control.mixins
+
+
+viewsets = {
+    # Note. This is the default Pulp Continer access policy with some modifications.
+    # Our changes have been marked with comments.
+    "distributions/container/container": {
+        "statements": [
+            {
+                "action": ["list"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": ["catalog"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": ["create"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_perms:container.add_containerdistribution",
+            },
+            {
+                "action": ["retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": [
+                    "has_namespace_or_obj_perms:container.view_containerdistribution",
+                ],
+            },
+            {
+                "action": ["pull"],
```

Comment:
there are 2 policies for pull operation - should you want to distinguish rules between private and public repo - it makes sense to keep this as it is proposed, otherwise it could be consolidated into one rule where the authentication is required regardless of the private/public repo type

```
{
                "action": ["pull"],
                "principal": "authenticated",
                "effect": "allow",
            },
```